server/src/main/java/io/spine/server/ServerEnvironment.java

```diff
@@ -532,7 +419,7 @@ public TypeConfigurator useTransportFactory(Fn<TransportFactory> fn) {
         /**
          * Assigns the specified {@code TransportFactory} for the selected environment.
```

Comment:
```suggestion
         * Assigns the specified {@code StorageFactory} for the selected environment.
```

Reply:
`StorageFactory` is configured starting from line no.439.

Reply:
Yes, you were right. Line numbers shifted after refactoring. Fixed.